code/modules/projectiles/guns/projectile/constructable/gunsmithing.dm

```diff
@@ -316,6 +363,20 @@
 			desc = "It looks like it could be some type of gun. It seems to be missing an ignition source."
 			icon_state = "stock_reservoir_barrel"
 			w_class = W_CLASS_LARGE
+
+		if("stock_crank_assembly")
+			name = "gun stock"
+			desc = "A metal gun stock. There is a crank charger loosely fitted to it."
+			icon_state = "stock_crank_assembly"
+		if("stock_crank")
+			name = "gun stock"
+			desc = "A metal gun stock. It seems to be missing a triggering mechanism."
+			icon_state = "stock_crank_assembly"
```

Comment:
wrong icon here? this and above share the same icon but you seem to have 2 available step icons in the pic in OP

Reply:
correct icon, stock_crank just represents the stage between securing the crank and adding a trigger. the second step icon is used below for lasmusket_assembly